storage/postgres/client.go

```diff
@@ -322,6 +326,7 @@ func (c *Client) Wipe(ctx context.Context) error {
 
 	// List, then drop all custom types.
 	// Query from https://stackoverflow.com/questions/3660787/how-to-list-custom-types-using-postgres-information-schema
+	// TODO: Don't delete extensions' types.
```

Comment:
Is this really a TODO, now that we're dropping the extension?

Reply:
we can switch course to dropping all extensions (except for some)

Reply:
Oh I see, so IIUC it's a TODO in the sense that if we added another extension, and that extension defined a custom type, and we chose not to use `DROP EXTENSION IF EXISTS` on it, we'd have to amend this code here so that it doesn't delete the extension's types.
In that hypothetical world, the same goes for other objects I guess? I don't know enough about postgres extensions but I imagine they can introduce functions and tables and whatnot.

Maybe this line can be something a little milder? Like `NOTE: If you introduce postgres extensions with their own custom types, this will try to delete those types too.`

I don't see it as a realistic TODO, i.e. I don't expect us to ever need to work on it in a principled way.